ccm_web/ecosystem.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     name: 'CCM - NestJS app',
     script: 'server/src/main.js',
     exec_mode: 'cluster',
-    instances: 'max',
+    instances: process.env.PM2_INSTANCES || 'max',
```

Comment:
I think it make sense to have `max` set locally since it will speed up the req/res locally. But shouldn't we default what were configuring on Prod instance?

Reply:
That's an interesting question. I guess I like having to set it again in prod so that it is made more explicit. It's also more challenging to keep things up to date if we match with prod, since we may have to change prod (that's the whole reason we make it configurable).

Reply:
A lot of our defaults in the project cater to local development, so I think this seems in line with those.

Reply:
ok. I really don't have a strong opinion here, but I just felt if there is problem we could reproduce earlier locally. But that might not always be true with memory/CPU stuff since we cannot map exact Prod setup. 

I am ok to keep the configuration as is.